app/controllers/admin/volunteers.py

```diff
@@ -97,3 +98,11 @@ def removeVolunteerFromEvent(user, eventID):
     (EventRsvp.delete().where(EventRsvp.user==user)).execute()
     flash("Volunteer successfully removed", "success")
     return ""
+
+@admin_bp.route('/updateBackgroundCheck', methods = ['POST'])
+def updateBackgroundCheck():
+    eventData = request.form
+    user = eventData['user']
+    setTo = int(eventData['setTo'])
+    updateOrCreateVolunteerBackground(user,setTo)
```

Comment:
instead of `setTo` I'd call it what it means, not what you are going to do with it. Like 'checkPassed`. Also, instead of casting it to an `int`, cast it to a `bool`

Reply:
whenever, I cast the data into a boolean, it data is set to always true. Left it as an int instead.